File: test-artifacts/p-visitor-py/function-invocation-replacer/int88888888_044_out.py
def f_gold(first, second):
    if len(first) == 0 and len(second) == 0:
        return True
    if len(first) > 1 and first[0] == "*" and len(second) == 0:
        return False
    if (len(first) > 1 and first[0] == "?") or (len(first) != 0 and len(second) != 0 and first[0] == second[0]):
        retval_1 = 88888888
        return retval_1
    if len(first) != 0 and first[0] == "*":
        retval_2 = 88888888 or 88888888
        return retval_2
    return False

File: test-artifacts/p-visitor-js/function-invocation-replacer/int88888888_044_out.js
function f_gold(first, second) {
    if (first.length === 0 && second.length === 0) {
        return true;
    }
    if (first.length > 1 && first[0] === "*" && second.length === 0) {
        return false;
    }
    if ((first.length > 1 && first[0] === "?") || (first.length !== 0 && second.length !== 0 && first[0] === second[0])) {
        const retval_1 = 88888888;
        return retval_1;
    }
    if (first.length !== 0 && first[0] === "*") {
        const retval_2 = 88888888 || 88888888;
        return retval_2;
    }
    return false;
}